cmd/nintendo-switch-checker/main.go

```diff
@@ -5,17 +5,22 @@ import (
 	"fmt"
 	"log"
 	"os"
+	"strings"
 	"sync"
 	"time"
 
+	"net/http"
+
 	"github.com/haya14busa/nintendo-switch-checker/nschecker"
 	"github.com/nlopes/slack"
 )
 
 var (
-	interval = flag.Duration("interval", 1*time.Minute, "Check interval")
-	channel  = flag.String("channel", "", "Slack channel name where checker posts comments")
-	once     = flag.Bool("once", false, "Check once")
+	interval     = flag.Duration("interval", 1*time.Minute, "Check interval")
+	slackToken   = flag.String("slack-token", "", "Slack API Token")
+	slackChannel = flag.String("slack-channel", "", "Slack channel name where checker posts comments")
+	lineToken    = flag.String("line-token", "", "LINE Notify Token https://notify-bot.line.me")
```

Comment:
Instead of specifying token with cli flag, please keep use env var.

I thinks we can add -notifier flags to specify notifier. (-notifier=slack or line)